classes/pvroot-image.bbclass

```diff
@@ -13,7 +13,7 @@ PVROOT_CONTAINERS_CORE ?= ""
 PVROOT_IMAGE_BSP ?= ""
 PVROOT_IMAGE ?= "yes"
 
-DEPENDS += " pvr-native squashfs-tools-native"
+DEPENDS += " pvr-native squashfs-tools-native ${INITRAMFS_IMAGE}"
```

Comment:
this feels like a left over from debugging?

Reply:
I'm not exactly sure if this is necessary, but I did run into one case where the inird image was missing...

Reply:
where is INITRAMFS_IMAGE set for you?